app/helpers/application_helper.rb

```diff
@@ -385,6 +385,12 @@ def show_parent?(obj)
     (obj.new_record? && obj.class.count > 0) || (!obj.new_record? && obj.class.count > 1)
   end
 
+  def documentation_button(section)
+    link_to(icon_text('question-sign', _('Documentation'), :class => 'icon-white'),
+            "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html##{section}",
```

Comment:
So what prevents us to build documentation for nightly when we bump Foreman version? Should I put it on some wiki? Somehow having not working buttons (even in nightly) does not seem right to me.